### DISCUSSION TITLE
Circle/oval hitbox

### DISCUSSION THREAD

**Discussion by Zimquadery** [4ian/GDevelop#2358] [Enhancement ideas (and their technical discussions)]
Circle or oval shaped hitbox would be good feature to add on gdevlop. It's so useful for top-down games. Using polygon to make a circle looking shape is annoying.Use cases : 
1. Circle or oval shaped enemy bosses 
2. Planets in space shooter games

**Comment by 4ian** [comment]
It's true that it could be useful, though it would probably be "pseudo circles" automatically made by GDevelop. I've made a card here: https://trello.com/c/JBRbTZLJ/476-add-support-for-circle-or-pseudo-circles-in-collision-masks
